tests/storage/test_data_import_cron.py

```diff
@@ -205,6 +208,63 @@ def second_object_cleanup(
     resource_class(namespace=namespace.name, name=second_object_name).clean_up()
 
 
+@pytest.fixture()
+def data_import_cron_pvc_source(
+    unprivileged_client,
+    data_import_cron_namespace,
+    storage_class_with_filesystem_volume_mode,
+    pvc_for_source_data_import_cron,
+):
+    # cron_template_spec = golden_images_data_import_cron_spec.template.spec
+    with DataImportCron(
+        name="pvc-import-cron",
+        namespace=data_import_cron_namespace.name,
+        schedule="*/1 * * * *",
+        managed_data_source="pvc-import-datasource",
+        annotations=BIND_IMMEDIATE_ANNOTATION,
+        template={
+            "spec": {
+                "source": {
+                    "pvc": {
+                        "name": pvc_for_source_data_import_cron.name,
+                        "namespace": pvc_for_source_data_import_cron.namespace,
+                    }
+                },
+                "sourceFormat": "pvc",
+                "volumeMode": "Filesystem",
```

Comment:
is it essential to have "volumeMode": "Filesystem",?

Reply:
no, and there's no `volumeMode` and `sourceFormat` field `DataImportCron.spec.template.spec`